spec/spec.md

```diff
@@ -113,14 +113,26 @@ Each interaction will `POST` specific data to the `callbackUrl`. But all `POST`s
 
 #### Reponse
 
-The `POST` to the provided `callbackUrl` can return with a simple successful HTTP response or it can return a `redirectUrl` that the app will open. This could be used to show a success message or bring them back to the website/app to continue where they left off. Most of the time `redirectUrl` will only be used when the user is already using their phone (see [above](#qr-code-or-link)).
+The `POST` to the provided `callbackUrl` can return with a simple successful HTTP response or it can return a success with follow up details. A `redirectLink` that the app will open in a browser or `challengeToken` that will start a new interaction.
```

Comment:
can you give an example of a new interaction?

Reply:
Yeah it would be a JWT string for any interaction https://specs.bloom.co/#challenge-token